pom.xml

```diff
@@ -81,6 +81,7 @@
         <!-- Sonarcloud properties -->
 
         <version.slf4j>1.7.30</version.slf4j>
+        <version.software-amazon-awssdk>2.14.12</version.software-amazon-awssdk>
```

Comment:
Would you mind using dots instead of hyphens here? Thanks! :+1:

Reply:
Done.